volatility3/framework/plugins/windows/dumpfiles.py

```diff
@@ -318,6 +320,7 @@ def _generator(self, procs: List, offsets: List):
                         )
 
         elif offsets:
+
             # Now process any offsets explicitly requested by the user.
             for offset, is_virtual in offsets:
                 try:
```

Comment:
The code below this will re-lookup the layers for each offset, far better to pick out the physical and virtual layers first (assuming we can identify what those are, there should be a `FIXME` next to the memory_layer bit to say "only works with intel and should be fixed once we choose a specific means for finding physical layers"...

Reply:
Adding this into the upcoming push too. Final test running now